src/components/policy/policy_external/src/update_status_manager.cc

```diff
@@ -184,14 +190,20 @@ void UpdateStatusManager::DoTransition() {
 
   current_status_ = next_status_;
   next_status_.reset();
-  listener_->OnUpdateStatusChanged(current_status_->get_status_string());
+
+  LOG4CXX_DEBUG(logger_, "last_processed_event_ = " << last_processed_event_);
+  if (last_processed_event_ != kScheduleManualUpdate) {
+    listener_->OnUpdateStatusChanged(current_status_->get_status_string());
+  }
```

Comment:
@jacobkeeler this method is redundant. Policy table should be saved to file system on shutdown any way. 
And during ignition cycle SDL use update_status manager for checking current state.